whatToDad_site/whatToDad_app/templates/post_action.html

```diff
@@ -0,0 +1,27 @@
+{% extends 'base.html' %} {% block content %}
+
+<div class="container">
+  <div class="row">
+    <div class="col-md-8 card mb-4  mt-3 left  top">
+      <div class="card-body">
+        <h1>{% block title %} {{ post.title }} {% endblock title %}</h1>
+        <p class=" text-muted">{{ post.author }} | {{ post.created_on }}</p>
+        <p class="card-text ">{{ post.content | safe }}</p>
```

Comment:
Using `safe` here is a bad idea. It allows users to inject arbitrary code into your site. If someone makes a malicious post they can inject arbitrary html onto the page and not just text